src/glint.gleam

```diff
@@ -820,17 +852,25 @@ fn command_help_to_usage_string(help: CommandHelp, config: Config) -> String {
     option.map(help.unnamed_args, args_count_to_usage_string)
     |> option.unwrap("[ ARGS ]")
 
+  // The max width of the usage accounts for the one tab indent plus two spaces
+  // of indentation on wrapped lines
+  let max_usage_width = config.max_output_width - tab_width - 2
```

Comment:
🔧 Probably want to use the configured column gap here

Reply:
Updated for custom indent width